<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonOpenQMaZdaFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open QMaZda Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpenImageFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenRoiFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open ROI Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMaZdaFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/QMaZda19_08_beta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/CTquality/DCMSel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditROIFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/CTquality/Roi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenOptionsFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Options Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOptionsFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/CTquality/Opt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenMzFeaturesOutFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Mz Feat. Out Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMzFeaturesOutFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/CTquality/Features/Gabor/l26f3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutFilePrefix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CT_l26f3_</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutFilePrefix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Out File Prefix</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>351</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOutBat">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>610</y>
      <width>411</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenBatFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenBatFolder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBatFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/CTquality</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBatFileName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaborl26f3</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBatFileName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bat File Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageFilePattern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.+l26.+.dcm</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImageFilePattern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image File pattern</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditImageFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenClassyfiersFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Classyfiers Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClassyfiersFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/CTquality/Classyfier/Gabor/l26f3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenPredictorOutputFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Pred. Out Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPredictorOutputFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/CTquality/PredictOut/Gabor/l26f3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenClassyfiersOptionsFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Class. Opt. File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClassyfiersOptionsFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/QMaZda19_08_beta/ClassOptions.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelROIFilePattern">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ROI File pattern</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditROIFilePattern">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.+.tif</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditROIFiles">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOptionsFilePattern">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>.+Gabor.+.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOptionsFilePattern">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Options File pattern</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOptionsFiles">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreateBatFile">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Create *.bat file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearOut">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Out</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCreateFeaturesBat">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Features Bat</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCreateTrainingBat">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Training Bat</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCreatePredictorBat">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Predictor Bat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReload">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
